src/argilla/client/feedback/schemas/records.py

```diff
@@ -251,3 +252,32 @@ def to_server_payload(self, question_name_to_id: Optional[Dict[str, UUID]] = Non
         if self.external_id:
             payload["external_id"] = self.external_id
         return payload
+
+
+class SortBy(BaseModel):
+    field: Union[str, RecordSortField]
+    order: Union[str, SortOrder] = SortOrder.asc
+
+    @validator("field")
+    def check_field_name(cls, field: Union[str, RecordSortField]):
+        if isinstance(field, str) and field.startswith("metadata."):
+            return field
+        elif isinstance(field, str):
+            return RecordSortField(field)
+        return field
```

Comment:
```suggestion
        return RecordSortField(field) if isinstance(field, str) else field
```

Reply:
RecordSortField is only for `inserted_at` and `updated_at` values. For metadata you should pass a `metadata.your_prop` string

Reply:
Then the code should look like the following, right?

```suggestion
        return RecordSortField(field) if isinstance(field, str) and not field.startswith("metadata.") else field
```

Reply:
What I mean is that if an `if-else-if` statement returns the same object inside one of the if-conditions than outside, it can be simplified as an `if-else` instead, in case you prefer that over the Python one-liner that can also be done as

```suggestion
        if isinstance(field, str) and not field.startswith("metadata."):
            return RecordSortField(field)
        return field
```